src/utilities/apibuilder/type/service.ts

```diff
@@ -144,16 +136,16 @@ export class ApiBuilderService {
     return this.schema.base_url;
   }
 
-  public findModelByName(name) {
-    return findTypeByName(this.models, name);
-  }
-
-  public findEnumByName(name) {
-    return findTypeByName(this.enums, name);
-  }
-
-  public findUnionByName(name) {
-    return findTypeByName(this.unions, name);
+  public findTypeByName(typeName: string) {
+    // By definition, a field or union type whose name is not fully qualified
+    // implies the type is defined internally, that is such type is not imported.
```

Comment:
While true, there is always the chance that that changes as the order is probably not enforced anywhere ;)

Reply:
the order of what is not enforced?

Reply:
That internal types preced external types in the list. Just musing about the assumption- not trying to block the change or anything :)